assets/js/pages/project/list.vue

```diff
@@ -54,8 +52,10 @@
                     if (index >= 0) {
                         this.projects.splice(index, 1)
                     }
+                    this.$store.dispatch("bearMessage/setMessage", { message: 'Project successfully removed', type: 'success' })
+
                 } catch (e) {
-                    //TODO: add snackbar
+                    this.$store.dispatch("bearMessage/setMessage", { message: 'Error during project remove...', type:  'error' })
```

Comment:
This action could be provided by some mixin as this will be used in many places.